src/main/java/eu/dissco/backend/domain/annotation/batch/AnnotationEvent.java

```diff
@@ -1,8 +1,9 @@
 package eu.dissco.backend.domain.annotation.batch;
 
-import eu.dissco.backend.domain.annotation.Annotation;
+import eu.dissco.backend.schema.Annotation;
 import java.util.List;
 
-public record AnnotationEvent(List<Annotation> annotations, List<BatchMetadata> batchMetadata) {
+public record AnnotationEvent(List<Annotation> annotationRequests,
+                              List<BatchMetadata> batchMetadata) {
```

Comment:
There's a batchmetadata schema but it's in the 0.1 directory, and not on schemas.dissco.tech. Updating those schemas are on the backlog, but i annotations/batchMetadata.json should be up to-date, just FYI